ChaosMod/Effects/db/Player/PlayerStun.cpp

```diff
@@ -0,0 +1,30 @@
+/*
+	Effect by OnlyRealNubs
+*/
+
+#include <stdafx.h>
+
+std::string effectName = "Dont_tazeme_bro";
+
+static void OnStart()
+{
+	Vector3 pos1 = GET_OFFSET_FROM_ENTITY_IN_WORLD_COORDS(PLAYER_PED_ID(), 0, 1, 0);
+	Vector3 pos2 = GET_ENTITY_COORDS(PLAYER_PED_ID(), 1);
+
+	Ped owner = GET_RANDOM_PED_AT_COORD(pos1.x, pos1.y, pos1.z, 50, 50, 50, -1);

```

Comment:
Why is this needed? You can just set "0" as the "owner" in "SHOOT_SINGLE_BULLET_BETWEEN_COORDS" (see the  Airstrike effect)

Reply:
Had no idea I could do that. Thanks for pointing that out.